bodhi/tests/server/test_bugs.py

```diff
@@ -274,20 +274,33 @@ def test_modified(self, info):
         bz._bz.getbug.return_value.product = 'aproduct'
         bz._bz.getbug.return_value.bug_status = 'NEW'
 
-        bz.modified(1411188)
+        bz.modified(1411188, 'A mean message.')
 
         bz._bz.getbug.assert_called_once_with(1411188)
         info.assert_called_once_with("Setting bug #1411188 status to MODIFIED")
         self.assertEqual(bz._bz.getbug.return_value.setstatus.call_count, 1)
 
+    @mock.patch.dict('bodhi.server.bugs.config', {'bz_products': 'aproduct'})
+    def test_modified_after_verified(self):
+        """Test the modified() method"""
+        bz = bugs.Bugzilla()
+        bz._bz = mock.MagicMock()
+        bz._bz.getbug.return_value.product = 'aproduct'
+        bz._bz.getbug.return_value.bug_status = 'VERIFIED'
+
+        bz.modified(1411188, 'A mean message.')
+
+        bz._bz.getbug.assert_called_once_with(1411188)
+        bz._bz.getbug.return_value.addcomment.assert_called_once_with('A mean message.')
+
     @mock.patch('bodhi.server.bugs.log.info')
     def test_modified_product_skipped(self, info):
         """Test the modified() method when the bug's product is not in the bz_products config."""
         bz = bugs.Bugzilla()
         bz._bz = mock.MagicMock()
         bz._bz.getbug.return_value.product = 'not fedora!'
 
-        bz.modified(1411188)
+        bz.modified(1411188, 'A mean message.')
 
         bz._bz.getbug.assert_called_once_with(1411188)
         info.assert_called_once_with("Skipping set modified on 'not fedora!' bug #1411188")
```

Comment:
I guess we could add an assert to check the comment here

Reply:
Here, it is testing the case when bug's product is not in bz_products and so it skips the functionality and returns. Hence, I believe, the comment assertion will not be possible because it is not getting executed.

Reply:
Sure let's keep this test as it is then :+1: